apps/web/server/lib/[user]/getServerSideProps.ts

```diff
@@ -166,7 +166,7 @@ export const getServerSideProps: GetServerSideProps<UserPageProps> = async (cont
     return {
       redirect: {
         permanent: false,
-        destination: `${urlDestination}?${urlQuery}`,
+        destination: `${encodeURI(urlDestination)}?${urlQuery}`,
```

Comment:
same here

Reply:
When user has 1 event-type, we redirect user to his/her only event-type booking page, and in this process, we use Next.js app router's `redirect` function, which uses non UTF-8 encoding standards, and hence we end up with a throw if the user's username contains special characters